airdrop/.gitignore

```diff
@@ -8,6 +8,7 @@ __pycache__/
 
 # Distribution / packaging
 .Python
+env/
```

Comment:
include `.env` and `.env.*` in the gitignore

Reply:
do we need dotEnv in python repo. 
If there is any tool being used by other Python repo, shall we reuse the same?

Reply:
No we are not using .env we used a template for gitignore from serverless community